data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -0,0 +1,441 @@
+            job_results = bq_utils.query(q=query_dict['query'], batch=True)
+            logging.info('%s table updated with clean rows in %s' %
+                         (query_dict['destination_table_id'],
+                          query_dict['destination_dataset_id']))
+            query_job_id = job_results['jobReference']['jobId']
+            query_job_ids.append(query_job_id)
+
+    incomplete_jobs = bq_utils.wait_on_jobs(query_job_ids)
+    if incomplete_jobs:
+        logging.info('Failed on {count} job ids {ids}'.format(
+            count=len(incomplete_jobs), ids=incomplete_jobs))
+        logging.info('Terminating retraction')
+        raise bq_utils.BigQueryJobWaitError(incomplete_jobs)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description=
+        'Runs retraction of deactivated EHR participants on all datasets in project. The rows to be retracted'
+        'are based on the date entered and will be retracted if the date entered is before the retraction date'
+        'Uses project_id, deactivated pid_table_id to determine '
+        'the pids to retract data for. The pid_table_id needs to contain '
+        'the person_id and research_id and deactivated_date columns specified in the schema above, '
+        'but research_id can be null if deid has not been run yet. ',
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument('-p',
+                        '--project_id',
+                        action='store',
+                        dest='project_id',
+                        help='Identifies the project to retract data from',
+                        required=True)
+    parser.add_argument('-t',
+                        '--ticket_number',
+                        action='store',
+                        dest='ticket_number',
+                        help='Ticket number to append to sandbox table names',
+                        required=True)
+    parser.add_argument(
+        '-pp',
+        '--pids_project_id',
+        action='store',
+        dest='pids_project_id',
+        help='Identifies the project where the pids table is stored',
+        required=True)
+    parser.add_argument(
+        '-pd',
+        '--pids_dataset_id',
+        action='store',
+        dest='pids_dataset_id',
+        help='Identifies the dataset where the pids table is stored',
+        required=True)
+    parser.add_argument('-pt',
+                        '--pids_table',
+                        action='store',
+                        dest='pids_table',
+                        help='Identifies the table where the pids are stored',
+                        required=True)
+    args = parser.parse_args()
+    query_list = create_queries(args.project_id, args.ticket_number,
+                                args.pids_project_id, args.pids_dataset_id,
+                                args.pids_table)
+    run_queries(query_list)
+    logging.info('Retraction complete')
```

Comment:
We need to address the logging.  You are importing the logging module, but you're not setting it up.  Because we don't have a logging module setup for the project, you must set up the logging yourself to use it (log file location(s), date formatting, message formatting, any rotating handlers) .  If you comment these lines:
```
#    query_list = create_queries(args.project_id, args.ticket_number,
#            args.pids_project_id, args.pids_dataset_id,
#            args.pids_table)
#    run_queries(query_list)
    logging.info('Retraction complete')
```
I believe you are expecting to see `Retraction complete` on your console but nothing is printed.  The deid runner sets up a logging location and console logger.  The function I'm thinking of is here:  https://github.com/all-of-us/curation/blob/develop/data_steward/tools/run_deid.py#L40 .  

Additionally, to identify your module's logger you should add this line:
```
LOGGER = logging.getLogger(__name__)
```

Then instead of calling `logging.info()` you would call `LOGGER.info()` because it has the logging instance for your module.